console/logging.rst

```diff
@@ -167,12 +167,13 @@ service configuration. Your method receives a
 :class:`Symfony\\Component\\Console\\Event\\ConsoleExceptionEvent` object,
 which has methods to get information about the event and the exception.
 
-Logging non-0 Exit Statuses
+Logging Error Exit Statuses
 ---------------------------
 
 The logging capabilities of the console can be further extended by logging
-non-0 exit statuses. This way you will know if a command had any errors, even
-if no exceptions were thrown.
+commands that return error exist statuses, which are any number different than
```

Comment:
The original text should have been "... exit ..."  instead of "... exist ..."  Would that be better?